### ISSUE TITLE
Flutter 1.20  Error: The non-abstract class 'ChipsInputState' is missing implementations for these members

### ISSUE THREAD

**Issue by stefensuhat** [danvick/flutter_form_builder#398]
After updating to Flutter 1.20 building the project return errors.

```
Error: The non-abstract class 'ChipsInputState' is missing implementations for these members:
 - TextInputClient.currentAutofillScope
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class ChipsInputState<T> extends State<ChipsInput<T>>
      ^^^^^^^^^^^^^^^
../../../flutter/packages/flutter/lib/src/services/text_input.dart:790:21: Context: 'TextInputClient.currentAutofillScope' is defined here.
  AutofillScope get currentAutofillScope;
```

**Comment by stefensuhat** [comment]
Put this on pubspec solved the problem.

`flutter_chips_input: ^1.9.0`

**Comment by awhitford** [comment]
Or run `flutter pub upgrade` to pull in these latest dependencies which were released today to coincide with the **Flutter 1.20** release.
